org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/Point.java

```diff
@@ -135,4 +135,23 @@ public int getIndex(String name) {
 		}
 		return indices;
 	}
+	
+	private static final String  VERTEX    = "([a-zA-Z0-9_])+\\((\\d+)\\)=([-+]?[0-9]*\\.?[0-9]+)";
+	private static final Pattern POSITION  = Pattern.compile(VERTEX+", "+VERTEX);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)